### ISSUE TITLE
Low Pri -  In the Tao Voting Module change "DELEGATED AND NON-DELEGATED VOTING" to something else

### ISSUE THREAD

**Issue by GriffGreen** [CommonsBuild/commons-config-frontend#50]
Can we make it one phrase like "Normal Vote Duration"

**Comment by divine-comedian** [comment]
I'll change it to Vote Duration since it is exactly the value of `Vote Duration` that the user inputs

**Comment by GriffGreen** [comment]
> I'll change it to Vote Duration since it is exactly the value of `Vote Duration` that the user inputs

Yeah but then with the extension... the Vote actually goes on for another day... so Vote Duration is a little weird...

**Comment by divine-comedian** [comment]
alright Griff, here ya go 

https://github.com/CommonsBuild/commons-config-frontend/commit/e79a4d17ef1bbad4511b4d09f97512919ee4bf79
